store/datastore.go

```diff
@@ -60,6 +60,20 @@ type DataStore interface {
 	// Updates device group
 	UpdateDeviceGroup(ctx context.Context, devid model.DeviceID, group model.GroupName) error
 
+	// UpdateDevicesGroup updates multiple devices' group, returning number
+	// of matching devices, the number devices that changed group and error,
+	// if any.
+	UpdateDevicesGroup(
+		ctx context.Context, devIDs []model.DeviceID, group model.GroupName,
```

Comment:
is this formatting on purpose? (same below)

Reply:
Yeah, I think I've developed some kind of wide-line-allergy. Especially for lines that wraps around my screen.